pages/docs/manual/latest/api/belt/array.mdx

```diff
@@ -765,7 +765,7 @@ let reduceWithIndex: (array<'a>, 'b, ('b, 'a, int) => 'b) => 'b
 
 `reduceWithIndex(xs, f)`
 
-Applies `f` to each element of `xs` from beginning to end. Function `f` has three parameters: the item from the array and an “accumulator”, which starts with a value of `init` and the index of each element. `reduceWithIndex` returns the final value of the accumulator.
+Applies `f` to each element of `xs` from beginning to end. Function `f` has three parameters: an “accumulator”, which starts with a value of `init` and the item from the array and the index of each element. `reduceWithIndex` returns the final value of the accumulator.
```

Comment:
```suggestion
Applies `f` to each element of `arr` from beginning to end. Function `f` has three parameters: an "accumulator", which starts with a value of `init` and the item from the array and the index of each element. `reduceWithIndex` returns the final value of the accumulator.
```

Reply:
xs -> arr looks much better while reading the docs